openapi-v1.yaml

```diff
@@ -6706,6 +6706,40 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /invitations/group/{invitation}/{namespace}/{group_name}/share:
```

Comment:
Let's make {group_name} -> {group}

Reply:
I am good since we have {group_name} everywhere, we should come back here and change though all path params